### DISCUSSION TITLE
MX Servers for Premium Customs Domain owners

### DISCUSSION THREAD

**Discussion by xyxzxyz** [simple-login/app#1962] [Feature Requests]
Recently I am facing quite a few sites that rejected SL's mx servers when using custom domains. I have to create email addresses using my customs domain pointing to PM's servers and those email addresses were accepted immediately. I hope SL would add MX servers only limited to premium users, perhaps pointing MX to Proton's mail servers.

**Comment by Wolf1098** [comment]
I have my proton account registering the same domain, without removing the sl records, putting the proton secondary, with catch-all enabled on proton. it works as a fail-over, can even initiate conversations within proton and it gets a sl alias record on the reply

**Reply by Wolf1098** [reply]
provided it didn't fail over to proton servers
